deploy/k8s/charts/trustification/values.schema.yaml

```diff
@@ -1140,9 +1150,8 @@ definitions:
       - disable
       - require
 
-  GuacMigrateDatabaseConfig:
+  UserPassConfig:
```

Comment:
Just an effort to save us from another identical configuration definition. 

BTW. this is still work in progress, I need to make more changes as it will not work as it currently stands.

Reply:
Ok. My concern here is that it might now look similar, but change in the future. So having two structs for two different concerns seems fine to me.

Reply:
I ended up, reverting it as it didn't work as expected. Simple properties is all we need here